controllers/main_controller.py

```diff
@@ -285,20 +262,16 @@ def __init__(self, *args, **kw):
         self._cache_expiration = 60 * 60 * 24
 
     def _render(self, *args, **kw):
-        kickoff_datetime_utc = datetime.datetime.strptime(
```

Comment:
Note that - we pull this `kickoff_datetime` variable for both the kickoff and offseason template. But in the kickoff template, we're (properly) treating it as an EST datetime, and in the offseason template we're treating it as a UTC datetime. We don't store these values with timezones either, which makes datetime logic weird.

Anyways - this feels like too much scope for a single controller to be doing, which is why I've pulled all of this datetime logic out in to the `SeasonHelper`

Reply:
uh, this is clowny and probably my fault :(